bindings/pyroot_experimental/pythonizations/test/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@
 # For the licensing terms see $ROOTSYS/LICENSE.
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
+# Test library loads during importing ROOT
+if (NOT APPLE OR NOT WIN32)
```

Comment:
I think this should be an 'AND'?

Reply:
Ah damn it :see_no_evil: yep, we see it failing below.

Reply:
Fixed